screens/product-list/product-list.css

```diff
@@ -0,0 +1,36 @@
+.filter {
+  display: flex;
+  align-items: center;
+}
+.filter button {
```

Comment:
Just a suggestion, be careful using element selectors as every button element in ```filter``` will get these styles.

Reply:
Sure will fix this!